### PR TITLE
Fix item state of flashlights & cult mirror shield

### PR DESCRIPTION
**Pull request by dj-34**
## What Does This PR Do
Fixes item states

## Why It's Good For The Game
No visual bugs

## Images of changes
image
image

## Testing
Tested in game

## Changelog
:cl:
fix: Sprite "on" item states on flashlights & cult mirror shield in hands
/:cl:

### CODE REVIEW

**Review by Henri215 — APPROVED** [review]
Lewcs good to me

File: code/game/objects/items/devices/flashlight.dm [hunk 1]
```diff
@@ -20,8 +20,10 @@
 /obj/item/flashlight/update_icon_state()
 	if(on)
 		icon_state = "[initial(icon_state)]-on"
+		item_state = "[initial(item_state)]-on"
 	else
 		icon_state = "[initial(icon_state)]"
+		item_state = "[initial(item_state)]"
```
Comment (Henri215) [inline code/game/objects/items/devices/flashlight.dm:26]: This causes flashlights that dont have a -on item_state to go invisible, you should probably scope it to only those that have it.

**Review by Henri215 — CHANGES_REQUESTED** [review]
Yeah :)

**Comment by dj-34** [comment]
fixed

**Review by S34NW — APPROVED** [review]

**Pull request merged**